public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -154,40 +164,46 @@ export class DashboardSettings extends PureComponent<Props> {
             <span>{dashboard.title} / Settings</span>
           </div>
         </div>
-        <div className="dashboard-settings__body">
-          <aside className="dashboard-settings__aside">
-            {pages.map((page) => (
-              <a
-                className={cx('dashboard-settings__nav-item', { active: page.id === editview })}
-                aria-label={selectors.pages.Dashboard.Settings.General.sectionItems(page.title)}
-                onClick={() => this.onChangePage(page.id)}
-                key={page.id}
-              >
-                <Icon name={page.icon} style={{ marginRight: '4px' }} />
-                {page.title}
-              </a>
-            ))}
-            <div className="dashboard-settings__aside-actions">
-              {canSave && <SaveDashboardButton dashboard={dashboard} onSaveSuccess={this.onPostSave} />}
-              {canSaveAs && (
-                <SaveDashboardAsButton dashboard={dashboard} onSaveSuccess={this.onPostSave} variant="secondary" />
-              )}
-            </div>
-          </aside>
-          <div className="dashboard-settings__scroll">
-            <CustomScrollbar autoHeightMin="100%">
+        <CustomScrollbar>
+          <div className={styles.scrollInner}>
+            <div className={styles.settingsWrapper}>
+              <aside className="dashboard-settings__aside">
+                {pages.map((page) => (
+                  <a
+                    className={cx('dashboard-settings__nav-item', { active: page.id === editview })}
+                    aria-label={selectors.pages.Dashboard.Settings.General.sectionItems(page.title)}
+                    onClick={() => this.onChangePage(page.id)}
+                    key={page.id}
+                  >
+                    <Icon name={page.icon} style={{ marginRight: '4px' }} />
+                    {page.title}
+                  </a>
+                ))}
+                <div className="dashboard-settings__aside-actions">
+                  {canSave && <SaveDashboardButton dashboard={dashboard} onSaveSuccess={this.onPostSave} />}
+                  {canSaveAs && (
+                    <SaveDashboardAsButton dashboard={dashboard} onSaveSuccess={this.onPostSave} variant="secondary" />
+                  )}
+                </div>
+              </aside>
               <div className="dashboard-settings__content">{currentPage.render()}</div>
-            </CustomScrollbar>
+            </div>
           </div>
-        </div>
+        </CustomScrollbar>
       </div>
     );
   }
 }
 
-export interface SettingsPage {
-  id: string;
-  title: string;
-  icon: IconName;
-  render: () => React.ReactNode;
-}
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  scrollInner: css`
+    min-width: 100%;
+    min-height: 100%;
+  `,
+  settingsWrapper: css`
+    background: ${theme.palette.gray10};
```

Comment:
this cannot be right, (feels like it will break in dark or light theme depending on what you tested). Use theme.colors.bg1 or bg2),  the palette colors should almost never be used from components

Reply:
Thanks @torkelo. Have swapped it out for `theme.colors.bg1`. 👍